src/views/workflow-history/workflow-history-event-status-badge/helpers/get-badge-icon-size.ts

```diff
@@ -0,0 +1,12 @@
+import type { Theme } from 'baseui';
+
+import { type WorkflowEventStatusBadgeSize } from '../workflow-history-event-status-badge.types';
+
+export default function getBadgeIconSize(
```

Comment:
```suggestion
export default function getBadgeIconSizeMap(
```
nit

Reply:
Instead, you can also pass this helper function the badge size and then return a size value based on that